README.md

```diff
@@ -30,6 +30,22 @@ Currently we are targeting five platforms: OS X, Ubuntu Linux, iOS, Android, and
 
 ## platforms ##
 
+### Windows (command line) ###
+Downloading angle https://www.dropbox.com/s/vollhh52bjemeym/angle.7z?dl=0 and extract to `build` directory
```

Comment:
Unless a trusted source provides verifiable downloads of these build artifacts, we need to provide instructions for users to build ANGLE on their machines.